stacker/config/__init__.py

```diff
@@ -355,10 +356,12 @@ def validate(self):
 
     def validate_stacks(self, data, value):
         if value:
-            names = set()
-            for i, stack in enumerate(value):
-                if stack.name in names:
+            stack_names = map(attrgetter('name'), value)
+            if len(set(stack_names)) == len(stack_names):
```

Comment:
Is returning None a valid return value for schematics validations?  I think to make this a little more clear, you might want to just pull the loop below into the opposite of this if statement.  ie:

```
if len(set(stack_names)) != len(stack_names):
    for i, stack_name in ... etc
```

Reply:
Python returns `None` implicitly when a function or method exits without a return.

Reply:
fixed